modules/user_accounts/php/edit_user.class.inc

```diff
@@ -454,12 +455,10 @@ class Edit_User extends \NDB_Form
             $user    = \User::factory($this->identifier);
             $success = $user->update($set);
         }
-        // prepend two random characters
-        if (isset($set['Password_hash'])) {
-            // Update CouchDB. Must do before password is salted/hashed.
-            $expiry = isset($values['Password_expiry'])
-                ? $values['Password_expiry'] : null;
-            $user->updatePassword($set['Password_hash'], $expiry);
+        // Now set the password. Note that this field is named incorrectly
+        // and represents a plaintext password, not a hash.
+        if (isset($values['Password_hash'])) {
+            $user->updatePassword($values['Password_hash']);
```

Comment:
what happened to Password_expiry? (Same with below.)

Reply:
There is no `Password_expiry` in the HTML of these pages. I believe they were copy-pasted from elsewhere once upon a time. Think that's also why there are references to CouchDB.